components/Footer.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+import Navigation from "@/components/Navigation";
+
+const StyledFooter = styled.footer`
+  background-color: #79af6e;
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 10vh;
+  z-index: 1000;
```

Comment:
Such a z-index should acutally never be necessary, if you place the the footer als last element in your _app.js, it should automatically be on top of the stacking context

Reply:
what he says

Reply:
Removed all z-indexes.